src/main/java/com/mailjet/client/transactional/response/MessageResult.java

```diff
@@ -0,0 +1,44 @@
+package com.mailjet.client.transactional.response;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class MessageResult {
+
+    /**
+     * Indicates the sending status of the message.
+     * Possible values:
+     * success
+     * error
+     */
+    private String status;
+
+    /**
+     * List containing information about any errors with the processing of this message.
+     * Will be displayed only if processing errors occur.
```

Comment:
What do you mean with "displayed"? Will the field be `null` if there is no processing error?

Reply:
Yep, will be null. Added XML doc with this info